### ISSUE TITLE
Compute managers shutting down early

### ISSUE THREAD

**Issue by peastman** [MolSSI/QCFractal#784]
I'm seeing a lot of cases of compute managers shutting down.  The logs report a server issue:

```
[2023-10-26 12:43:37 PDT]     INFO: ComputeManager: Executor local_executor has 0 active tasks and 3 open slots
[2023-10-26 12:43:37 PDT]  WARNING: ComputeManager: Acquisition of new tasks failed: Unable to refresh JWT authorization token! This is a server issue!!
[2023-10-26 12:44:03 PDT]  WARNING: ComputeManager: Heartbeat failed: Unable to refresh JWT authorization token! This is a server issue!!. QCFractal server down?
[2023-10-26 12:44:03 PDT]  WARNING: ComputeManager: Missed 6 heartbeats so far
[2023-10-26 12:44:03 PDT]  WARNING: ComputeManager: Too many failed heartbeats, shutting down.
```

What could be causing this?

**Comment by bennybp** [comment]
This was a bug on my part. I just pushed a fix this afternoon when I saw the same thing  (#783).

I will increase the refresh token time in the meantime, which will help until the next release (which I want to make pretty soon)

**Comment by bennybp** [comment]
I did just increase the refresh token time, but currently-running managers may see this if they run longer than 24 hours. Newly-created managers should be good for a month.

**Comment by peastman** [comment]
Thanks!  Since the longest job allowed in any queue is seven days, I think a month should be plenty.

**Comment by bennybp** [comment]
Fixed and released in v0.52
